pkg/apis/awsprovider/awsprovider.go

```diff
@@ -0,0 +1 @@
+package awsprovider
```

Comment:
Should we not just leave the `group.go` file in place?

Reply:
does the file name matter if we keep the package name the same?

Reply:
Not really, I was just trying to reduce the diff in this PR, this is minor